ufo2fontir/src/source.rs

```diff
@@ -163,10 +165,7 @@ impl Source for DesignSpaceIrSource {
             let ufo_dir = self.designspace_dir.join(&source.filename);
 
             // TODO less dumbed down version of dim => location mapping
-            let mut location = DesignSpaceLocation::new();
-            source.location.iter().for_each(|dim| {
-                location.insert(dim.name.clone(), dim.xvalue.map(|v| v as i32).unwrap());
-            });
+            let location = to_ir_location(&source.location);
```

Comment:
I think if you define From this could be .into()?

https://doc.rust-lang.org/std/convert/trait.From.html

Reply:
I can't `impl From<&[Dimension]> for ir::DesignSpaceLocation` because ir::DesignSpaceLocation is just an alias to BTreeMap at the moment, and also it belongs to a different crate (fontir vs ufo2fontir), plus slices are always foreign

Reply:
Oh right ... no traits across boundaries